services/user.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: Optional[str] = None,
+    first_name: Optional[str] = None,
+    last_name: Optional[str] = None
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:

```

Comment:
you are refferring to build-in User model here but we have redefined our  own in models